app/controllers/api/v3/facilities_controller.rb

```diff
@@ -26,7 +26,7 @@ def disable_audit_logs?
   def transform_to_response(facility)
     Api::V3::FacilityTransformer
       .to_response(facility)
-      .merge(sync_region_id: sync_region_id(facility))
+      .merge(sync_region_id: facility.block_region_id)
```

Comment:
Should this be a config? For small districts, we could sync the full district right? It can also be different for different countries. I don't know if this fits within the gamut of this PR though.

Reply:
As of now we block sync everywhere. Moving to config would be easy when the need arises.

Reply:
It very well could be, but I agree - this probably isn't the PR to make that change 😄

Reply:
cool 👌